src/node.cc

```diff
@@ -3334,6 +3338,13 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "_debugWaitConnect", True(env->isolate()));
   }
 
+  // --expose_internals, --expose-internals
+  // Note that this is not exposed as a process property, it is deleted when
```

Comment:
I want confused. I'm not fond of this pattern. Exposing via process.binding('config') is preferable to me.